psqlgraph/node.py

```diff
@@ -240,13 +250,12 @@ def to_json(self):
 
     @classmethod
     def from_json(cls, node_json):
-        if cls is Node:
-            Type = Node.get_subclass(node_json['label'])
-            if not Type:
-                raise KeyError('Node has no subclass named {}'
-                                  .format(node_json['label']))
-        else:
-            Type = cls
+        abstract_node_cls = cls.get_edge_class().get_node_class()
```

Comment:
is there a reason you need to jump via an edge class here?

Reply:
`from_json` can be called from some class say `Aliquot`, which could be from the `Node` base class or some namespaced base class. Whichever it is, had to figure out the right base class to use. The `get_edge_class` will always return the right `Edge` class, which in turn will return the right `Node` class